README.md

```diff
@@ -24,6 +25,29 @@ There are different levels to this assignment, level 1 being the easiest, level
 
 _Warning: this endpoint has gotchas_
 
+## Assignment 2: Autonomous Car
+
+### Instructions
+
+* Download and run this server (see [Getting Started](#getting-started))
+* Build an application / tool in React or Python with the following criteria:
+    * Fetches the a autonomous car route from this API server
+    * Takes a set of instructions as input
+    * Determine's if the route was a success or a failure
+    * If it was a failure, determine at which step or position that failed
+* There are different levels to this assignment, level 1 being the easiest, level 2 having some gotchas, etc. If you complete a level with ease, feel free to jump to the next one. Please timebox your solution to one day total.
+
+### Level 1
```

Comment:
These levels relate to the tax-assignment

Reply:
Whoops! Should be fixed now.